### DISCUSSION TITLE
Branch Management

### DISCUSSION THREAD

**Discussion by lyxal** [Vyxal/Vyxal#264] [Q&A] [answered]
So apparently constantly PRing a single branch isn't a good idea. What should we do instead?

**Comment by Bubbler-4** [comment, marked as answer]
You can always do this:

- Create a feature branch "feature1" and open a PR to its parent "parent"
- Work on "feature1"
- When you want to merge the PR, do it, delete the current "feature1", and create a fresh "feature1" branch again (if the same name causes problems then you can add numbering like "feature1-1", "feature1-2", ...)
- Continue working on "feature1"...
